lightwood/analysis/model_analyzer.py

```diff
@@ -50,7 +50,7 @@ def model_analyzer(
     data_type = dtype_dict[target]
     data_subtype = data_type
 
-    is_numerical = data_type in [dtype.integer, dtype.float] or data_type in [dtype.array]
+    is_numerical = data_type in [dtype.integer, dtype.float, dtype.array, dtype.tsarray]
```

Comment:
is there a reason this is a "[]" as opposed to "()"; earlier convention had tuples but this wont change anything